src/messages/MessageBuilder.cpp

```diff
@@ -28,6 +28,14 @@ MessagePtr makeSystemMessage(const QString &text, const QTime &time)
     return MessageBuilder(systemMessage, text, time).release();
 }
 
+EmotePtr makeAutoModBadge()
+{
+    return std::make_shared<Emote>(Emote{
+        EmoteName{}, ImageSet{Image::fromPixmap(getResources().twitch.automod)},
+        Tooltip{"AutoMod"},
+        Url{"https://dashboard.twitch.tv/settings/moderation/automod"}});
```

Comment:
This actually redirects to, e.g. `https://dashboard.twitch.tv/u/zneix/settings/moderation/automod`, which might be useful if we'll add something like `jump to "badge's url"` in the future